packages/modules/cra-template-desktop-viewer/template/src/react-app-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
i dont think you need this file, should get auto generated when you create a new app using our cra template and fork

Reply:
Mistake copy. I deleted